src/lib/icons/DetailedSpinner.tsx

```diff
@@ -0,0 +1,57 @@
+import styled, { keyframes } from 'lib/theme'
+
+const rotate = keyframes`
+from {
+  transform: rotate(0deg);
+}
+to {
+  transform: rotate(360deg);
+}
+`
+
+const DetailedSpinnerWrapper = styled.div`
+  align-items: center;
+  display: flex;
+  height: 1rem;
```

Comment:
Considering that the InnerDot and OuterRing are defined in terms of px, I think the height and width should be set to 16px and not 1rem.